docker-compose.yml

```diff
@@ -325,6 +346,9 @@ services:
       - AEROSPIKE_HOST=aerospike:3000
       - COUCHBASE_HOST=couchbase
       - COUCHBASE_PORT=8091
+      - AWS_LAMBDA_ENDPOINT=http://serverless-lambda:8080
+      - CONTAINER_HOSTNAME=http://integrationtests
+    hostname: integrationtests
```

Comment:
Is this necessary 🤔 Why do you need it out of interest?

Reply:
yes, see my other comment